extensions/amp-story-shopping/0.1/amp-story-shopping.js

```diff
@@ -41,7 +41,7 @@ export const loadFonts = (win, fontFaces) => {
 AMP.extension('amp-story-shopping', '0.1', (AMP) => {
   AMP.registerElement(
     'amp-story-shopping-tag',
-    AmpStoryShoppingTag,
+    dependsOnStoryServices(AmpStoryShoppingTag),
```

Comment:
Sync getters can be safely used by wrapping the implementation class in `dependsOnStoryServices`.